src/Components/CreateNewPoi/CreatePoiFourthPage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import validate from './validate';
+import poiFourthPagesFields from './PoiFourthPageFields';
+
+const CreatePoiFourthPage = (props) => {
+  const {
+    handleSubmit, pristine, submitting, previousPage,
+  } = props;
```

Comment:
Do you deconstruct your props this way in a functional component ?

Reply:
deconstruction changed.